tests/capice_resources/train_data_creator/test_train_data_creator.py

```diff
@@ -58,6 +60,46 @@ def test_classmethod_list_further_processing_columns(self):
             ]
         )
 
+    def subtest_vcf_header(self, vcf_header: list[str]) -> None:
+        """
+        Subtest to the component test of train-data-creator, to test if the VCF header contains
+        the proper information.
+        """
+        self.subtest_vcf_header_datetime(vcf_header)
+        self.subtest_vcf_header_version(vcf_header)
+        self.subtest_vcf_header_filenames(vcf_header)
+
+    def subtest_vcf_header_datetime(self, vcf_header: list[str]) -> None:
+        """
+        Test to ensure the date is set correctly.
+        Testing with today and tomorrow to prevent errors to be raised during midnight testing
+        """
+        today = datetime.now()
+        time_format = '%Y%m%d'
+        possible_dates = [
+            f'##fileDate={today.strftime(time_format)}',
+            f'##fileDate={(today + timedelta(days=1)).strftime(time_format)}'
+        ]
+        intersect = set(vcf_header).intersection(set(possible_dates))
+        self.assertEqual(len(intersect), 1)
+
+    def subtest_vcf_header_version(self, vcf_header: list[str]) -> None:
+        """
+        Test to ensure the CAPICE-resources metadata line is set correctly
+        """
+        self.assertIn(f'##CAPICE-Resources_version={__version__}', vcf_header)
+
+    def subtest_vcf_header_filenames(self, vcf_header: list[str]) -> None:
+        """
+        Test to ensure that proper filenames are set within the VCF metadata.
+        """
+        files = [
+            '##Used_VKGL_File=smol_vkgl.tsv.gz',
+            '##Used_ClinVar_File=smol_clinvar.vcf.gz'
+        ]
```

Comment:
Maybe better to enforce dates in files (and also test that input files have a date in them).